src/dto/ResponseDto.js

```diff
@@ -7,9 +7,8 @@ export default class ResponseDto {
      * @param {String} response - What to respond with or the image location.
      * @param {String} title - What to put as the title of the response.
      */
-    constructor(trigger = [''], isRegex = false, response = '', title = '') {
+    constructor(trigger = [''], response = '', title = '') {
```

Comment:
You need to update the places where ResponseDto is used, for example the autoresponse command still passes false here